flink/src/main/java/org/apache/iceberg/flink/IcebergSinkUtil.java

```diff
@@ -37,9 +45,32 @@
 import static org.apache.iceberg.TableProperties.WRITE_TARGET_FILE_SIZE_BYTES_DEFAULT;
 
 class IcebergSinkUtil {
+
   private IcebergSinkUtil() {
   }
 
+  @SuppressWarnings("unchecked")
+  static DataStreamSink<RowData> write(DataStream<RowData> inputStream,
+                                       Map<String, String> options,
+                                       Configuration conf,
+                                       String fullTableName,
+                                       Table table,
+                                       TableSchema requestedSchema) {
+    IcebergStreamWriter<RowData> streamWriter = createStreamWriter(table, requestedSchema);
```

Comment:
I just notice `IcebergStreamWriter` does not set chaining strategy. Should add in the constructor of `IcebergStreamWriter`: `setChainingStrategy(ChainingStrategy.ALWAYS);`